cmd/permit.go

```diff
@@ -0,0 +1,140 @@
+/*
+Copyright © 2020 Hirohito Sasakawa
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	bq "cloud.google.com/go/bigquery"
```

Comment:
The order of package import should be as follows;

https://github.com/golang/go/wiki/CodeReviewComments#imports

```suggestion
...
	"errors"
	"fmt"

	bq "cloud.google.com/go/bigquery"
	"github.com/spf13/cobra" 
```